pkg/frontend/querymiddleware/astmapper/instant_splitting.go

```diff
@@ -194,12 +197,20 @@ func (i *instantSplitter) mapCall(expr *parser.Call) (mapped parser.Expr, finish
 		return i.mapCallAvgOverTime(expr)
 	case countOverTime:
 		return i.mapCallVectorAggregation(expr, parser.SUM)
+	case increase:
+		return i.mapCallVectorAggregation(expr, parser.SUM)
+	case irate:
+		return i.mapCallRate(expr)
 	case maxOverTime:
 		return i.mapCallVectorAggregation(expr, parser.MAX)
 	case minOverTime:
 		return i.mapCallVectorAggregation(expr, parser.MIN)
+	case presentOverTime:
+		return i.mapCallVectorAggregation(expr, parser.MAX)
 	case rate:
 		return i.mapCallRate(expr)
+	case resets:
+		return i.mapCallVectorAggregation(expr, parser.SUM)
```

Comment:
Note to reviewers: read the [PR description](https://github.com/grafana/mimir/pull/2601#issue-1322340408) before reviewing the code.